### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by harivamsi9**
The previous git clone git@github.com:$username/PrairieLearn.git is deprecated. So updated with the latest working command.

And the current documentation is lacking the step change directory, so added it.

### CODE REVIEW

**Comment by jonatanschroeder** [comment]
Thank you for your contribution. The cd command makes sense, but afaict using SSH for cloning a repo is not deprecated. Can you clarify what you mean?

**Comment by jonatanschroeder** [comment]
That said, a link to https://docs.github.com/en/authentication/connecting-to-github-with-ssh might be a good idea.

**Comment by nwalters512** [comment]
@harivamsi9 do you intend to fix up this PR? Otherwise I can make the change for you and get this merge. As Jonatan points out, SSH git operations are definitely not deprecated. It'd be interesting to know where you read that they were.

**Comment by harivamsi9** [comment]
Hey @nwalters512 I intend to fix this. 

This is was error I was getting
image


When I searched on stackoverflow, https://stackoverflow.com/questions/21255438/git-permission-denied-publickey-fatal-could-not-read-from-remote-repository

Maybe I needed to setup the SSH key? I don't know. But if it is something that needed to be setup, then I believe it would be better if we have instructions for that as well. 

Otherwise, we can modify according to this PR.

**Comment by nwalters512** [comment]
Correct, you'll need to set up SSH keys first. The only reference to deprecation on that Stack Overflow page is in relation to password-based _HTTPS_ access. I agree with Jonatan's suggestion of including a link to https://docs.github.com/en/authentication/connecting-to-github-with-ssh.

**Comment by harivamsi9** [comment]
Awesome! I will make the required modifications. 

Just out of curiosity, 
I was thinking using git clone https://github.com/$user_name/PrairieLearn.git might be easy to get started for the users. Are they any technical reasons or benefits of using the git clone git@github.com:$user_name/PrairieLearn.git over this? 

Sorry if this question seems to be very basic.

**Comment by nwalters512** [comment]
Probably the only reason we picked SSH is because that's what all the PL core devs use. While cloning with HTTPS allows you to avoid authenticating at the beginning, you'll still need to auth as soon as you actually try to push something, and it's not clear to me that HTTPS auth is significantly easier to configure than SSH, especially with the existence of [`gh auth login`](https://cli.github.com/manual/gh_auth_login).

**Comment by harivamsi9** [comment]
@nwalters512 Can you please review now?

File: docs/contributing.md [hunk 1]
```diff
@@ -6,10 +6,13 @@
 
 **Getting started as a collaborator:** Everyone should fork the main repository on GitHub. The full name of this repository is `PrairieLearn/PrairieLearn` (`"organization_name/repo_name"`); when you fork it, GitHub will create a repo named `user_name/PrairieLearn`. If you're not sure how to fork the repo, see [Github's instructions](https://help.github.com/articles/fork-a-repo/).
 
+Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to github with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```
Comment (timdyang) [inline docs/contributing.md:9]:
```suggestion
Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```

Comment (tdy) [inline docs/contributing.md:9]:
```suggestion
Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```

File: docs/contributing.md [hunk 2]
```diff
@@ -6,10 +6,13 @@
 
 **Getting started as a collaborator:** Everyone should fork the main repository on GitHub. The full name of this repository is `PrairieLearn/PrairieLearn` (`"organization_name/repo_name"`); when you fork it, GitHub will create a repo named `user_name/PrairieLearn`. If you're not sure how to fork the repo, see [Github's instructions](https://help.github.com/articles/fork-a-repo/).
 
+Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```
Comment (jonatanschroeder) [inline docs/contributing.md:9]:
```suggestion
Before you go further, you will need to set up your own SSH keys and associate them with your GitHub account (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```

**Review by jonatanschroeder — APPROVED** [review]

**Comment by nwalters512** [comment]
@harivamsi9 you'll need to accept the CLA as described here: https://github.com/PrairieLearn/PrairieLearn/pull/8832#issuecomment-1807756324

**Comment by harivamsi9** [comment]
I have reviewed and hereby sign the CLA

**Pull request merged**